package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "image-picker",
+  "version": "0.3.0",
```

Comment:
Good point - I've changed it to `0.3.1` so it could potentially be merged with a future patch release.